apps/server/prisma/seed.ts

```diff
@@ -0,0 +1,188 @@
+import { PrismaClient } from '@prisma/client';
+import { v4 as uuidv4 } from 'uuid';
+
+const prisma = new PrismaClient();
+
+const LEADERSHIP_TEAM_UUID = uuidv4();
```

Comment:
it might be better if we didn't generate these uuids, because we wouldn't have consistency across local instances and it might make debugging and stuff more difficult if they kept changing. maybe we can just use hardcoded strings?